readyplayerme/rpm-unity-sdk-core
# [SDK-301] Add runtime analytics to quick start

## [SDK-301](https://ready-player-me.atlassian.net/browse/SDK-301)

## Description

- Added scripts for runtime analytics
- Added new analytic events for quick start scene
- Restructure/renaming for few analytics files
- Analytics for Integration guide

## How to Test

- Enable analytics 
- Run quick start scene and load personal avatar
- Open integration guide and try few documentation links
- Observe events on amplitude



## Checklist








[SDK-301]: https://ready-player-me.atlassian.net/browse/SDK-301?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
